department-of-veterans-affairs/caseflow
# Updates to EditCavcRemandForm and Jest Tests

Connects [CASEFLOW-1052](https://vajira.max.gov/browse/CASEFLOW-1052)

### Description
This is a follow-up to earlier work on 1052 that adjusts the logic for handling the fields for `decisionType` and `remandType` based on the mode (`add` or `edit`) of the component.

This also adds back in Jest tests for `EditCavcRemandForm` that were removed in order to get earlier changes merged.

### Acceptance Criteria
- [x] `EditCavcRemandForm` works as it did before in edit mode
- [x] Jest tests for `EditCavcRemandForm` are present

### Testing Plan
1. View the "editing" stories in Storybook, ensure that forms work as they did prior
2. View in Storybook, ensure the default ("add") mode works correctly
3. Verify that editing a CAVC remand with this branch behaves the same way it does in `master`


### User Facing Changes
Should look the same
